content/manager_architecture/metrics-flow.md

```diff
@@ -21,30 +21,40 @@ You can send metrics to the management environment using any transport (agent),
 
 RabbitMQ stores metrics within a metrics-dedicated, non-durable, non-exclusive topic exchange.
```

Comment:
RabbitMQ stores metrics within a metrics-dedicated, non-durable, non-exclusive topic exchange. After a metric is consumed it is removed from the queue.
If you remove the proprietary consumer, you can connect an external system to the RabbitMQ to consume metrics, but this is not an officially supported implementation.

Reply:
fixed